### ISSUE TITLE
License plate detection - Pass the plate number to HA

### ISSUE THREAD

**Issue by mikedahlgren** [AngellusMortis/pyunifiprotect#304]
Feature request.  Currently the License plate detection for the AI cameras(AI Bullet and AI 360 Panoramic) shows the license plate number overlaid on the video and send via the ubiquity app notification.  In Home Assistant the notification is just a sensor with a binary "License Plate Detected" Yes/No.  

I do not know if it is possible to pull this information, however it would be extremely useful to have the plate number inside of Home Assistant.

**Comment by AngellusMortis** [comment]
It is not a binary sensor in HA. It is a sensor that has the license plate number.

**Comment by mikedahlgren** [comment]
Thank you so much for your quick reply!  I promise I spent a lot of time working on this and completely missed the license plate coming through for other reasons.  

Also thank you for the License plate blueprint, this has been one of those killer features I've been wanting since I started working with home Assistant and I appreciate you making it possible!
(https://community.home-assistant.io/t/unifi-protect-blueprints/362604)
